internal/cmd/exodus.go

```diff
@@ -178,6 +179,12 @@ func exodusMain(ctx context.Context, cfg conf.Config, args args.Config) int {
 	for _, item := range items {
 		gwItem := gw.ItemInput{WebURI: webURI(item.SrcPath, args.Src, destTree, srcIsDir)}
 
+		// Try to detect MIME type of file leaving content-type unset if we can't.
+		mtype, err := mimetype.DetectFile(item.SrcPath)
+		if err == nil {
```

Comment:
Could we please ensure that a non-nil 'err' ends up being logged, at least at Debug level, so we have a possibility of debugging it if content types are not being detected correctly?

Note, if you don't want to introduce a new code path for that (to keep tests simpler) you could just always do a log.Debug after the call to DetectFile, containing mtype, err and SrcPath, regardless of whether there was an error.

Reply:
Yeah, that sounds good. We actually don't need to prevent using `mtype` anyway, as `mimetype.DetectFile` returns a valid object (with type application/octet-stream) even when it encounters an error (i.e., IO).